<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Guild Wars 2</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_general">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Currently, there are no general settings available yet.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_advanced">
    <attribute name="title">
     <string>Advanced</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBox_onlineStateTransmissionThreshold">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_locationTransmissionThreshold">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_onlineStateTransmissionThreshold">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Online state transmission threshold</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_locationTransmissionThreshold">
        <property name="text">
         <string>Location timeout transmission threshold</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_locationTransmissionThreshold_2">
        <property name="text">
         <string>seconds</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_onlineStateTransmissionThreshold_2">
        <property name="text">
         <string>seconds</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_distanceTransmissionThreshold">
        <property name="text">
         <string>Location distance transmission threshold</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_distanceTransmissionThreshold_2">
        <property name="text">
         <string>meters</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_distanceTransmissionThreshold">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>gridLayoutWidget</zorder>
    <zorder>label_onlineStateTransmissionThreshold</zorder>
   </widget>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>buttonBox</zorder>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
